### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dailymp**
Adding linter ESLINT configs and configuring Circle CI pipelines for testing and linter, also CD has added to ghpages

### CODE REVIEW

File: 03 Tests samples/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+- Tests samples for events onChange ( simulate, fireEvent, ...) (Regina)
+- Tests samples for events onClick (Aurora)
+- Tests samples for events onScroll (Daily)
+- Test samples for onMouseUp and onMouseDown (Regina)
+- Test samples for onContextMenu (Regina)
+- Test samples for onSubmit (Aurora)
+
+- Tests samples for useSuscribeToScroll (Daily)
+- Tests samples using ref an useRef hook (Regina, Daily)
```
Comment (reginafernandezpinar) [inline 03 Tests samples/README.md:27]: What about removing the names here once the issues have been created for each person?

File: 03 Tests samples/src/app.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import { MessagesSection } from './components';
+
+export const App: React.FunctionComponent = () => (
+	<div>
+		<MessagesSection />
+	</div>
```
Comment (mathewamg33) [inline 03 Tests samples/src/app.tsx]: Could we change the indentation to 2 spaces?

**Review by mathewamg33 — APPROVED** [review]

File: 03 Tests samples/README.md (hunk 1 above)
Comment (dailymp) [reply 03 Tests samples/README.md:27]: Yes, done, removed now, thanks!

File: 03 Tests samples/src/app.tsx (hunk 2 above)
Comment (dailymp) [reply 03 Tests samples/src/app.tsx]: Ok, done I have changed the rule, and changed all code to meet the new rule

Comment (dailymp) [reply 03 Tests samples/src/app.tsx]: > Could we change the indentation to 2 spaces?

Ok, done I have changed the rule, and changed all code to meet the new rule

File: 03 Tests samples/README.md (hunk 1 above)
Comment (dailymp) [reply 03 Tests samples/README.md]: > What about removing the names here once the issues have been created for each person?

Yes, done, removed now, thanks!

**Comment by dailymp** [comment]
Comments from reviewers fixed and ready to merge

**Pull request merged**